webpack/development.config.js

```diff
@@ -0,0 +1,49 @@
+const webpack = require('webpack');
+const { assetsDir } = require('../config');
+const baseWebpackConfig = require('./base.config');
+
+module.exports = Object.assign({}, baseWebpackConfig, {
+	entry: [
+		'webpack-dev-server/client?http://localhost:3000',
+		'webpack/hot/only-dev-server',
+	].concat(baseWebpackConfig.entry),
```

Comment:
Probably worth using webpack-merge (https://github.com/survivejs/webpack-merge).

Reply:
How would it improve current config? I don't see enough value in adding extra dependency that does the same thing that we could achieve with plain javascript.

Reply:
Well, in my opinion it could have been done in more elegant way with webpack-merge. But using plain Javascript makes sense as well.